Samples/Chirper/ChirperGrains/ChirperAccount.cs

```diff
@@ -31,7 +31,6 @@ public class ChirperAccountState
         public string UserAlias { get; set; }
     }
 
-    [Reentrant]
```

Comment:
Why do you think it should not be Reentrant?
I am not 100% sure, but I vaguely recall there might have been some cycles in the call graph,  maybe 2 users trying to follow each other in parallel, so it had to be reentrant to avoid deadlocks.
@jthelin will know better.